libs/file/Makefile

```diff
@@ -8,28 +8,27 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=file
-PKG_VERSION:=5.33
-PKG_RELEASE:=2
+PKG_VERSION:=5.34
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://sources.lede-project.org/ \
```

Comment:
Remove sources as PKG_SOURCE_URL, it's already in the fallback mechanism
Fedora URL --> HTTPS